README.md

```diff
@@ -123,6 +123,19 @@ cp /usr/i686-w64-mingw32/bin/lib{gcc_s_dw2-1,ssp-0,stdc++-6,winpthread-1}.dll /m
 wine /my/path/to/Winamp/winamp.exe
 ```
 
+## Building & Running on Windows
+
+Don't. But if you must there are to ways to build
+
+1. open the folder with Visual Studio
+2. use the generate_project_files.bat script, which produces a 32-bit only Visual Studio project using CMake.
+
+If you don't want to do either, but use CMake in some other way here are some caveats to be aware of:
+
+* CMake GUI doesn't work and doesn't tell you why it doesn't work.
+* CMake doesn't work very well if you don't clean out the generated files first. Visual Studio handles this for you.
+* Nothing builds under 64-bit with the MS compiler due to __asm blocks
+* There are crashes in evallib using the latest compilers, which rules out pretty much any AVS preset from working (currently)
```

Comment:
This shouldn't be in this section, because it affects all MSVC builds. And just say: "Currently, MSVC builds will not run most presets, due to EEL bugs" or something quick and neutral like this.